app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', counter_cache: :posts_counter
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  # def last_5_comments
```

Comment:
As part of the requirement, The Post model should have a method that returns the 5 most recent comments for a given post. Please create a method in `post.rb` that will show the 5 most recent comments. 👍🏻

Reply:
Thanks 👍🏻 

I completely forgot to do that.

:heavy_check_mark: Now it's done